event_ticket_maxlimit/models/event_ticket.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class EventTicket(models.Model):
+    _inherit = "event.event.ticket"
+
+    tickets_per_registration = fields.Integer(
+        string="Tickets per Registration",
+        help="Number of tickets per registration. 0 means unlimited.",
```

Comment:
```suggestion
        help="Number of tickets per registration.",
```

Keeping a default value of 1 or 4 is more meaningful. Selling unlimited tickets on one user is not a proper business logic

Reply:
The default value is already set to  9 ticket per registration.